plutus-tx/src/PlutusTx/Builtins/Internal.hs

```diff
@@ -262,7 +262,7 @@ ripemd_160 (BuiltinByteString b) = BuiltinByteString $ Hash.ripemd_160 b
 
 verifyEd25519Signature :: BuiltinByteString -> BuiltinByteString -> BuiltinByteString -> BuiltinBool
 verifyEd25519Signature (BuiltinByteString vk) (BuiltinByteString msg) (BuiltinByteString sig) =
-  case PlutusCore.Crypto.Ed25519.verifyEd25519Signature_V1 vk msg sig of
+  case PlutusCore.Crypto.Ed25519.verifyEd25519Signature vk msg sig of
```

Comment:
Presumably this should have been using the V2 variant (I'm not sure what we do about variants in Plinth), but it doesn't matter now anyway.

Reply:
This only matters if one runs Plinth code as Haskell (rather than in the CEK machine).